azimuth/modules/base_classes/module.py

```diff
@@ -223,7 +224,7 @@ def get_current_pipeline(self) -> PipelineDefinition:
                 " as it does not use ModelContractScope."
             )
         if self.config.pipelines is None:
-            raise ValueError("A model was not provided in the config.")
+            raise ValueError("A pipeline was not provided in the config.")
```

Comment:
While we're at it, I'd suggest something like
```suggestion
            raise ValueError("No pipelines were provided in the config.")
```
or even
```suggestion
            raise ValueError("No pipelines configured.")
```